telemetry/specs/INT.mdk

```diff
@@ -948,38 +963,16 @@ INT Metadata Header and Metadata Stack:
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 `
 
-* INT metadata header is 8 bytes long followed by a stack of INT metadata.
+* INT metadata header is 12 bytes long followed by a stack of INT metadata.
   Each metadata is either 4 bytes or 8 bytes in length. Each INT hop adds
   the same length of metadata. The total length of the metadata stack is
   variable as different packets may traverse different paths and hence
   different number of INT hops.
 
 * The fields in the INT metadata header are interpreted the following way:
-  - Ver (4b): INT metadata header version. Should be 1 for this version.
-  - Rep (2b): Replication requested. Support for this request is optional. If
-this value is non-zero, the device may replicate the INT packet. This is useful
-to explore all the valid physical forwarding paths when multi-path forwarding
-techniques (e.g., ECMP, LAG) are used in the network. Note the Rep bits should
-be used judiciously (e.g., only for probe packets, not for every data packet).
-While we recommend that Rep bits be set only for probe packets, the INT
-architecture does not (and perhaps cannot) disallow use of the Rep bits for real
-data packets.
-    - 0: No replication requested.
-    - 1: Port-level (L2-level) replication requested. If the INT packet is
-forwarded through a logical port that is a port-channel (LAG), then replicate
-the packet on each physical port in the port-channel and send a single copy per
-physical port.
-    - 2: Next-hop-level (L3-level) replication requested. Forward the packet
-to each L3 ECMP next-hop valid for the destination address, with INT headers
-replicated in each forwarded copy.
-    - 3: Port-level and Next-hop-level replication requested.
-  - C (1b): Copy.
-    - If replication is requested for data packets, the INT Sink must be
-able to distinguish the original packet from replicas so that it can forward
-only original packets up the protocol stack, and drop all the replicas. The C
-bit must be set to 1 on each copy, whenever an INT hop replicates a packet.
-The original packet must have C bit set to 0.
-    - C bit must be set to 0 in the original packet by INT source
+  - Ver (4b): INT metadata header version. Should be 2 for this version.
+  - Res (2b): Reserved
+  - C   (1b): Copy/Clone. INT Sink should Discard the packet after Extracting INT data 
```

Comment:
I wonder about calling this "Copy/Clone". Is a probe packet a copy or a clone?
In the IETF I suggested calling it "active".
If you have any better suggestions, I am open to hearing them.

Reply:
how about D for Discard?
Sink should discard the pkt (after generating a report) no matter what the reason is.

Reply:
Works for me

Reply:
Works for me as well. Changed it to "D" in the doc.